bench.project.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Benchmark Suite",
+    "tree": {
+        "$className": "DataModel",
+        "StarterPlayer": {
+            "$className": "StarterPlayer",
+            "StarterPlayerScripts": {
+                "$className": "StarterPlayerScripts",
+                "$path": "tests"
+            }
+        },
+        "ReplicatedStorage": {
+            "$className": "ReplicatedStorage",
+            "$path": "DevPackages",
+            "Matter": {
+                "$path": "lib"
+            },
+            "PinnedMatter": {
+                "$path": "PinnedMatter.rbxm"
+            },
+            "Jecs": {
+                "$path": "PinnedJecs.luau"
```

Comment:
Luau specified but the file is an actual rbxm in the repository. I do agree it should be a lua file that we can easily change rather than going for rbxms. But paths doesnt accept modules, only folders. So you would have to do `: "PinnedJecs/src"`